op-supervisor/supervisor/backend/cross/safe_start.go

```diff
@@ -77,6 +79,10 @@ func CrossSafeHazards(d SafeStartDeps, chainID eth.ChainID, inL1Source eth.Block
 				return nil, fmt.Errorf("msg %s was included in block %s derived from %s which is not in cross-safe scope %s: %w",
 					msg, includedIn, initSource, inL1Source, types.ErrOutOfScope)
 			}
+			// Run expiry window invariant check *after* verifying that the message is non-conflicting.
```

Comment:
We run this check at the end to defer unnecessary block replacements until we're back in-scope.

Reply:
Yes, such executing messages are invalid.
I interpreted conflicts as an executing message that cannot be corroborated from local view, which is slightly different here. But re-reading the commentary for `ErrConflict`, I now see that the error is already appropriate:
> 	// ErrConflict happens when we know for sure that there is different canonical data
I'll revise this to use `ErrConflict` rather than `ErrExpired`.